tests/load-tests/ci-scripts/collect-results.sh

```diff
@@ -6,228 +6,150 @@ set -o pipefail
-+ $csv_delim_quoted + (.metadata.creationTimestamp) \
-+ $csv_delim_quoted + (.status.startTime) \
-+ $csv_delim_quoted + (.status.finallyStartTime) \
-+ $csv_delim_quoted + (.status.completionTime) \
-+ $csv_delim_quoted + (if .status.startTime != null and .metadata.creationTimestamp != null then ((.status.startTime | strptime($dt_format) | mktime) - (.metadata.creationTimestamp | strptime($dt_format) | mktime) | tostring) else \"\" end) \
-+ $csv_delim_quoted + (if .status.finallyStartTime != null and .status.startTime != null then ((.status.finallyStartTime | strptime($dt_format) | mktime) - (.status.startTime | strptime($dt_format) | mktime) | tostring) else \"\" end) \
-+ $csv_delim_quoted + (if .status.completionTime != null and .status.finallyStartTime != null then ((.status.completionTime | strptime($dt_format) | mktime) - (.status.finallyStartTime | strptime($dt_format) | mktime) | tostring) else \"\" end) \
-+ $csv_delim_quoted + (if .status.conditions[0].status == \"True\" and .status.completionTime != null and .metadata.creationTimestamp != null then ((.status.completionTime | strptime($dt_format) | mktime) - (.metadata.creationTimestamp | strptime($dt_format) | mktime) | tostring) else \"\" end) \
-+ $csv_delim_quoted + (if .status.conditions[0].status == \"False\" and .status.completionTime != null and .metadata.creationTimestamp != null then ((.status.completionTime | strptime($dt_format) | mktime) - (.metadata.creationTimestamp | strptime($dt_format) | mktime) | tostring) else \"\" end)"
-oc get pipelineruns.tekton.dev -A -o json >"$pipelineruns_json"
-jq "$jq_cmd" "$pipelineruns_json" | sed -e "s/\n//g" -e "s/^\"//g" -e "s/\"$//g" -e "s/Z;/;/g" | sort -t ";" -k 13 -r -n >>"$pipelinerun_timestamps"
-
-## Application service log segments per user app
-echo "Collecting application service log segments per user app..."
-oc logs -l "control-plane=controller-manager" --tail=-1 -n application-service >"$application_service_log"
-mkdir -p "$application_service_log_segments"
-for i in $(grep -Eo "${USER_PREFIX}-....-app" "$application_service_log" | sort | uniq); do grep "$i" "$application_service_log" >"$application_service_log_segments/$i.log"; done
-## Error summary
-echo "Error summary:"
-if [ -f "$load_test_log" ]; then
-    grep -Eo "Error #[0-9]+" "$load_test_log" | sort | uniq | while read -r i; do
-        echo -n " - $i: "
-        grep -c "$i" "$load_test_log"
-    done | sort -V || :
-else
-    echo "WARNING: File $load_test_log not found!"
-fi
-
-## Monitoring data
-echo "Setting up tool to collect monitoring data..."
+echo "[$(date --utc -Ins)] Setting up Python venv"
+{
 python3 -m venv venv
 set +u
 source venv/bin/activate
 set -u
 python3 -m pip install -U pip
 python3 -m pip install -e "git+https://github.com/redhat-performance/opl.git#egg=opl-rhcloud-perf-team-core&subdirectory=core"
+python3 -m pip install tabulate
+python3 -m pip install matplotlib
```

Comment:
```suggestion
python3 -m pip install matplotlib
python3 -m pip install 'tenacity<8.4.0'
```

Recent changes to tenacity module cause failues - see https://github.com/jd/tenacity/issues/471